### PR TITLE
Update services

### PR DESCRIPTION
**Pull request by christhomas**
I made changes to how the services are rendered in the page, the styles and the html, so that its more subtle and easier to read on mobile and desktop like pages.

### CODE REVIEW

**Comment by christhomas** [comment]
Nothing changed here is complicated or weird, but the problem I am finding is that the more pull requests I do, the more conflicts I have to fix manually.

Merging conflicts in GIT is basically an artform, you just learn it.  What has to happen, is that you instinctively know that when given a conflict, the right code that you want to have at the end of merging each problematic block of code.

This comes from basically having a good memory of what you had in the past, and what you had in this commit and knowing how to choose, is it left, or right? It's not something you can be taught.  Apart from learning the grammer of what constitutes a merge conflict, but this is a mere technicality.

The skill of knowing, "This is old code, so I dont want it" and "This is new code, so I do want it", and also "This is a mix of old and new, dunno why, stupid git, but I KNOW HOW TO FIX IT".

All comes from practice and making mistakes and realising afterwards how to fix it.

Remember that Git allows you to compare against previous versions, so if you have merged something badly, you can do a comparison with your broken code and a version you know that was not broken and then manually try to patch it up back into a working state.

Oh this will be fun :D

File: index.php [hunk 1]
```diff
@@ -30,10 +30,10 @@
         break;
 
 // Service Details Pages
-    case preg_match("~^services$~", $url):
+    case preg_match("~^[/]?services$~", $url):
```
Comment (christhomas) [inline index.php]: This small "[/]?" might not seem like it does much, but remember that I said regular expressions were black magic that barely anybody truly understands? ok, so I will try to explain what this tiny piece of text does.

[] - means character to match, whatever you put inside here, the engine will try to match against the input text

so I put [/] because I want to match the / character, in our situation, we have [/]?services, so I want to match "services" or "/services", you see thats what it is doing, its saying, I want to search for the / character, hence [/] is an expression, you can put multiple characters inside, like [a-z] which means any character between a to z, or [a-zA-Z] which means any a to z character, in lower or upper case

the []? means, whatever characters I search for, I want ZERO OR MORE of them, in other words, zero matches would mean, I want to match it, or not, I just want to see if the / character is there and if it is present, great! match it, if it is not there, still match it, as long as the "services" part of the string is there, it will match

in other words, "[/]?services" means, I want to match "/services" or "services"

**Review by yakotta — DISMISSED** [review]
Press/testimonials/publicity sections appear to be broken
